src/System-Support.package/SystemVersion.class/instance/initialize.st

```diff
@@ -1,4 +1,10 @@
 initialization
 initialize
 	super initialize.
+	type := 'Pharo'.
+	major := 0.
+	minor := 0.	 	
+	suffix := ''.
+	build := -1. "means no build number given, as we also accept a build number 0"
```

Comment:
invalidBuildNumber should be used here. Then the comment is not required

Reply:
Fixed it